src/components/Expander/Expander.jsx

```diff
@@ -23,6 +21,20 @@ const Expander = (props) => {
 		...passThroughs
 	} = props;
 
+	Expander.Label = () => null;
```

Comment:
we wouldn't want to redefine all this stuff every render tho?

Reply:
True. Was just trying to get the tool to work. It fails if it's in a separate code block because the functions below cause errors.

```
  const labelChildProp = _.first(
     _.map(findTypes(props, Expander.Label), "props")
   );
   const additionalLabelContentChildProp = _.first(
     _.map(findTypes(props, Expander.AdditionalLabelContent), "props")
   );
```

I also can't put them above Expander in separate code blocks because then I'm attacking Label to Expander which doesn't exist yet.